doc/Caliptra.md

```diff
@@ -1098,9 +1118,9 @@ The PAUSER field of the APB interface is used to encode device attributes for th
 
 The Caliptra mailbox commands are specified in the [Caliptra runtime firmware specification](https://github.com/chipsalliance/caliptra-sw/blob/main/runtime/README.md#maibox-commands).
 
-### Hash calculation HW API
+### Hash calculation HW API (Subsystem mode only)
```

Comment:
why is this Subsystem only?

Reply:
In general, I think all of the crypto offload should be usable in both modes

Reply:
Because we removed exposing SHA directly over AXI-SUB to resolve two fundamental issues we faced during 1.x:

FIPS issue: someone can use that SHA pre-FIPS testing
Functional issue: Someone can lock that SHA not allowing FW to use it to accelerate its hash calc and is being fed block by block in 1.x

Further that capability was added for folks that wanted to keep Caliptra as the only security controller in the chip; so they can take subsystem for integration.

In addition to solving above problems, it reduces maintenance and validation of multiple configurations.

Reply:
"In general, I think all of the crypto offload should be usable in both modes" - unless AXI-MGR exists and you enable that support in passive mode, which we said we should not as a starting point, we will not have the same capabilities

Reply:
We should talk more about this as this is a feature regression.

Reply:
Yes, we can but my stance is anyone wanting to use Caliptra to do SOC level hashing, they can use subsystem mode. The feature was added for certain company that wanted to have Caliptra as a single security controller.

Reply:
I'm not sure how significant the performance regression would be, but in theory we could have a mailbox API for hashing.

In passive mode I do think there's still value in using Caliptra for crypto offload where possible since it makes the SoC iRoT simpler.

"unless AXI-MGR exists and you enable that support in passive mode, which we said we should not as a starting point during the guardrails, we will not have the same capabilities."

Is this a technical limitation (i.e. technically challenging to expose hash acc w/o AXI-MGR) or just a design decision (for the reasons you listed above). Trying to figure out if it's worth discussing those points or if there's a deeper technical issue.

Reply:
It might be possible with a bunch of FW configurations but we need SOCs to trust that Caliptra FW will not access anything on SOC in passive mode. We can talk more about this. Would you be OK if we bin this item and mark this comment resolved? 

@jhprabhu can merge the PR if you are fine with it

Reply:
I think @jhand2 gave a go...so marking this resolved